pages/pipelines/managing_log_output.md.erb

```diff
@@ -143,3 +143,72 @@ For example, if you have environment variable `MY_SECRET="topsecret"`and you run
 <h3 class="Docs__note__heading">Setting environment variables</h3>
 <p>Note that if you <emphasis>set</emphasis> or <emphasis>interpolate</emphasis> a secret environment variable in your <code>pipeline.yml</code> it is not redacted, but doing that is <a href="/docs/pipelines/secrets#anti-pattern-storing-secrets-in-your-pipeline-dot-yml">not recommended</a>.</p>
 </div>
+
+## Private build log archive storage
+
+By default, build logs are stored in encrypted form in Buildkite's Amazon S3 buckets. If you want to have an increased level of security, you can store the archived build logs in your private AWS S3 bucket.
+
+<div class="Docs__note">
+<p>You need to be subscribed to the <a href="https://buildkite.com/pricing">Buildkite Enterprise</a> payment plan to use a private AWS S3 build log archive location.</p>
+</div>
+
+To set up a private build log archive storage:
+
+1. Create an Amazon S3 bucket in *us-east-1* location (the only region currently supported).
+1. Provide read and write access permission policy for the Buildkite's AWS account `032379705303`. Here's an example policy:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AllowBuildkiteToWriteObjectsInLogsPrefix",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::032379705303:root"
+            },
+            "Action": "s3:PutObject",
+            "Resource": "arn:aws:s3:::my-bucket/my-prefix/*",
+            "Condition": {
+                "StringEquals": {
+                    "s3:x-amz-acl": "bucket-owner-full-control"
+                }
+            }
+        },
+        {
+            "Sid": "AllowBuildkiteToReadObjectsInLogsPrefix",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::032379705303:root"
+            },
+            "Action": "s3:GetObject",
+            "Resource": "arn:aws:s3:::my-bucket/my-prefix/*"
+        },
+        {
+            "Sid": "AllowBuildkiteToDeleteObjectsInLogsPrefix",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::032379705303:root"
+            },
+            "Action": "s3:DeleteObject",
+            "Resource": "arn:aws:s3:::my-bucket/my-prefix/*"
+        },
+        {
+            "Sid": "AllowBuildkiteToListBucketInLogsPrefix",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::032379705303:root"
+            },
+            "Action": "s3:ListBucket",
+            "Resource": "arn:aws:s3:::my-bucket",
+            "Condition": {
+                "StringLike": {
+                    "s3:prefix": "my-prefix/*"
+                }
+            }
+        }
+    ]
+}
+```
+
+1. Reach out to [success@buildkite.com](mailto:success@buildkite.com) to complete the setup. You will need to provide the information about your S3 bucket.
```

Comment:
Can we be specific about what into they need to supply? 

It would also be useful to tell folks which of that whole code sample they need to edit. and which are magic numbers they need to leave.

Reply:
@sj26 @nikwek could you please take a look at this PR & shed some light on @plaindocs' questions here?

Reply:
Once the customer provides us with the S3 bucket address our engineering team can continue the configuration on our side. 

The code block is the S3 bucket configuration with Buildkite's account number in it. I think the whole configuration is required for it to work.

Reply:
Thanks, @nikwek! Updating the PR.